rails/.acraconfigs/acra-server/encryptor.yaml

```diff
@@ -0,0 +1,62 @@
+defaults:
+  crypto_envelope: acrablock
+  reencrypting_to_acrablocks: true
+
+schemas:
+  - table: dependencies
+    columns:
+      - id
+      - requirements
+      - created_at
+      - updated_at
+      - rubygem_id
+      - version_id
+      - scope
+      - unresolved_name
+    encrypted:
+      - column: unresolved_name
+        data_type: bytes
+        # "Tm8gcGVybWlzc2lvbg==" == "No permission"
+        default_data_value: "Tm8gcGVybWlzc2lvbg=="
+
+  - table: versions
+    columns:
+      - id
+      - authors
+      - description
+      - "number"
+      - rubygem_id
+      - built_at
+      - updated_at
+      - summary
+      - platform
+      - created_at
+      - indexed
+      - prerelease
+      - "position"
+      - latest
+      - full_name
+      - size
+      - licenses
+      - requirements
+      - required_ruby_version
+      - sha256
+      - metadata
+      - yanked_at
+      - required_rubygems_version
+      - info_checksum
+      - yanked_info_checksum
+      - pusher_id
+      - cert_chain
+      - canonical_number
+    encrypted:
+    - column: authors
+      data_type: bytes
+      # "SGlkZGVuIGF1dGhvcg==" == "Hidden author"
```

Comment:
maybe if we meet such need by ourselves or at least 1-2 user's requests for that - we can discuss it. For now, imho, it will be overengineering due to `bytes` will be used rarely in comparison with other types. And we left it more for `default_value` feature. But, imho, not much client's need it and they will prefer app's types like `int`/`str`/etc. 
Plus, it complicates configuration. It requires new configuration language over our YAML configuration language over YAML format. For now it's exception for `bytes`. If we will provide `bytes`/`text`, it cause new questions in other parts `does this option supports same formatting for values?`, `do I need add "text:" for text value for this option?`, and so on.